sharding-core/src/main/java/io/shardingsphere/core/keygen/BuiltinKeyGeneratorType.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016-2018 shardingsphere.io.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * </p>
+ */
+
+package io.shardingsphere.core.keygen;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Built-in key generator type.
+ *
+ * @author panjuan
+ */
+@RequiredArgsConstructor
+@Getter
+public enum BuiltinKeyGeneratorType {
```

Comment:
We never use word for `Builtin` in our project, just reference master slave rule.
How about rename this class to `KeyGeneratorType` directly?